machine/netberg/netberg_common_rangeley/INSTALL

```diff
@@ -0,0 +1,224 @@
+
+Overview
+--------
+
+An "ONIE recovery image" is used for:
+
+1. installing ONIE on a blank machine, i.e. what manufacturing would
+   use.
+
+2. recovering a destroyed ONIE system.
+
+The recovery image is:
+
+. <machine>.iso       -- a hybrid ISO image.
+
+ISO Recovery Image
+------------------
+
+The .iso image is used to create the usb recovery disk.
+
+You can use the .iso image to create a bootable USB memory stick.
+Use ``dd`` to copy the .iso image to a USB stick and boot from that::
+
+  dd if=<machine>.iso of=/dev/sdX bs=10M
+
+For can find the correct ``/dev/sdX`` by inspecing the ``dmesg``
+output after inserting the USB stick into your work station.
+
+To enable booting from USB in the AMI-BIOS:
+
+1. Insert the USB stick
+
+2. Power on the Netberg Rangeley CPU board Switch
+
+3. While seeing ``Press <DEL> or <ESC> to enter setup.``, press ESC to interrupt
+  You shall see:
+
+       Aptio Setup Utility - Copyright (C) 2015 American Megatrends, Inc.       
+    Main  Advanced  IntelRCSetup  Server Mgmt  Security  Boot  Save & Exit      
+/----------------------------------------------------+-------------------------\
+|  BIOS Information                                  |Set the Date. Use Tab    |
+|  BIOS Vendor             American Megatrends       |to switch between Date   |
+|  Core Version            5.009                     |elements.                |
+|  Compliancy              UEFI 2.3; PI 1.2          |                         |
+|  Project Version         0HURA 1.42 x64            |                         |
+|  Build Date and Time     11/19/2015 17:23:17       |                         |
+|                                                    |                         |
+|  Memory Information                                |                         |
+|  Total Memory            8192 MB (DDR3)            |                         |
+|                                                    |-------------------------|
+|                                                    |><: Select Screen        |
+|  System Date             [Fri 05/26/2017]          |^v: Select Item          |
+|  System Time             [12:10:30]                |Enter: Select            |
+|                                                    |+/-: Change Opt.         |
+|  Access Level            Administrator             |F1: General Help         |
+|                                                    |F2: Previous Values      |
+|                                                    |F3: Optimized Defaults   |
+|                                                    |F4: Save & Exit          |
+|                                                    |ESC: Exit                |
+\----------------------------------------------------+-------------------------/
+        Version 2.17.1245. Copyright (C) 2015 American Megatrends, Inc.        
+                                                                             AB
+
```

Comment:
These "screen shots" of the BIOS menus are *super* helpful.  Thanks so much for adding that.